### PR TITLE
cli: fix incorrect actual values for constellation verify on AWS

### PR DESCRIPTION
**Pull request by 3u13r**
### Proposed change(s)

- Use the same struct and PCR extraction as in the aTLS verifier

verify, aws: https://github.com/edgelesssys/constellation/actions/runs/5925328580

### Checklist

- [x] Add labels (e.g., for changelog category)
- [x] Is PR title adequate for changelog?
- [x] Link to Milestone

### CODE REVIEW

**Comment by 3u13r** [comment]
I'm not sure about the additional dependency, Feel free to make suggestions.

File: cli/internal/cmd/verify.go [hunk 1]
```diff
@@ -372,17 +375,22 @@ func (f *attestationDocFormatterImpl) parseCerts(b *strings.Builder, certTypeNam
...
 	for pcrNum, expectedPCR := range expectedPCRs {
-		encPCR := quotes[1].Pcrs.Pcrs[fmt.Sprintf("%d", pcrNum)]
-		actualPCR, err := base64.StdEncoding.DecodeString(encPCR)
+
+		pcrIdx, err := vtpm.GetSHA256QuoteIndex(quotes)
+		if err != nil {
+			return fmt.Errorf("get SHA256 quote index: %w", err)
+		}
+
+		encPCR := quotes[pcrIdx].Pcrs.Pcrs[pcrNum]
```
Comment (katexochen) [inline cli/internal/cmd/verify.go:387]:
```suggestion
		actualPCR := quotes[pcrIdx].Pcrs.Pcrs[pcrNum]
```

File: cli/internal/cmd/verify.go [hunk 2]
```diff
@@ -372,17 +375,22 @@ func (f *attestationDocFormatterImpl) parseCerts(b *strings.Builder, certTypeNam
 }
 
 // parseQuotes parses the base64-encoded quotes and writes their details to the output builder.
-func (f *attestationDocFormatterImpl) parseQuotes(b *strings.Builder, quotes []quote, expectedPCRs measurements.M) error {
+func (f *attestationDocFormatterImpl) parseQuotes(b *strings.Builder, quotes []*tpmProto.Quote, expectedPCRs measurements.M) error {
 	writeIndentfln(b, 1, "Quote:")
 	for pcrNum, expectedPCR := range expectedPCRs {
-		encPCR := quotes[1].Pcrs.Pcrs[fmt.Sprintf("%d", pcrNum)]
-		actualPCR, err := base64.StdEncoding.DecodeString(encPCR)
+
```
Comment (katexochen) [inline cli/internal/cmd/verify.go:381]:
```suggestion
```

**Review by katexochen — APPROVED** [review]
lgtm otherwise

File: cli/internal/cmd/verify.go (hunk 1 above)
Comment (katexochen) [reply cli/internal/cmd/verify.go:387]: plz check if nil

**Review by daniel-weisse — APPROVED** [review]

**Pull request merged**